ui/lib/core/addon/templates/components/replication-secondary-card.hbs

```diff
@@ -15,40 +15,40 @@
     </h3>
     <div class="grid-item-left">
       {{#if (get (cluster-states this.state) "isOk")}}
-        <h6 class="title is-6">
+        <h4 class="title is-6">
```

Comment:
Whoops didn't realize I didn't comment on this file! If we can update the `h4` and `p` tags to be HDS text as well that would be amazing :D Thank you!!

Reply:
I think this component needs to be fully re-worked, because the heading tags don't really mean anything and the organization of the DOM doesn't really make any sense. That's out of scope for this PR, and I don't want us to skip it in the future because we think it has HDS text so it's been updating -- so, I'm going to skip. Thanks!